src/hostname/hostnamed.c

```diff
@@ -661,11 +680,11 @@ static int property_get_firmware_date(
                 void *userdata,
                 sd_bus_error *error) {
 
-        _cleanup_free_ char *firmware_date = NULL;
+        uint64_t firmware_date = 0;
```

Comment:
Please use `usec_t`. Also, I'd like to make the default value `USEC_INFINITY`, instead of `0`.